ui/packages/catalog-ui-search/src/main/webapp/react-component/container/sharing/sharing.tsx

```diff
@@ -255,6 +260,18 @@ export class Sharing extends React.Component<Props, State> {
     this.componentDidMount()
   }
 
+  unSubscribeUsers = () => {
+    this.usersToUnsubscribe.forEach(async user => {
```

Comment:
✏️ Would prefer user renamed to avoid confusion between this and the user imported  in this file